core/frontend/src/tile/TileAdmin.ts

```diff
@@ -161,6 +161,8 @@ export class TileAdmin {
   public readonly contextPreloadParentSkip: number;
   /** @beta */
   public readonly cesiumIonKey?: string;
+  /** @beta */
```

Comment:
Potential location for new conditional to use/not use cache, can definitely be moved if somewhere else is better.